fixtures/room/room_update_fixtures.py

```diff
@@ -0,0 +1,107 @@
+
+null = None
+
+query_update_all_fields = '''mutation{
+                updateRoom(name:"Entebbe",newName:"Jinja",capacity:8,roomType:"board room"){
```

Comment:
Avoid this name/new name logic, once id is supplied as a required positional argument, any other thing that is supplied is updated, any other field that is not supplied is left as was